prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -120,8 +120,9 @@ struct fi_ibv_rdm_tagged_rndv_header {
 	struct fi_ibv_rdm_header base;
 	uint64_t src_addr;
 	void *id;
-	int len;
+	size_t total_len;
```

Comment:
Variable sized fields really shouldn't be part of a wire protocol.  I.e. pointers and size_t, which differ based on platform. uint64_t is a better option.